### PR TITLE
SW-2530 Use new tooltip component to add tooltips to icon buttons

### PR DESCRIPTION
**Pull request by alexTerraformation**

### CODE REVIEW

File: src/components/seeds/database/index.tsx [hunk 1]
```diff
@@ -607,13 +607,15 @@ export default function Database(props: DatabaseProps): JSX.Element {
...
     <>
       <Box marginLeft={1} display='inline'>
-        <Button
-          id='more-options'
-          icon='menuVertical'
-          onClick={(event) => event && handleClick(event)}
-          priority='secondary'
-          size='medium'
-        />
+        <Tooltip title='More Options'>
```
Comment (karthikbtf) [inline src/components/seeds/database/index.tsx:610]: `title={strings.MORE_OPTIONS)` for i18n purposes? and in the other places..

**Review by karthikbtf — COMMENTED** [review]
Lgtm 👍 
Just one small requested change but a needed one..
This is a nice UX improvement, thanks 🎉

File: src/components/seeds/database/index.tsx (hunk 1 above)
Comment (alexTerraformation) [reply src/components/seeds/database/index.tsx:610]: Doh!  Of course.  Had that in my head to come back to and forgot.  Thanks for catching!

**Review by karthikbtf — APPROVED** [review]
👍

**Review by chudo76 — COMMENTED** [review]
Looks good; thank you!

**Pull request merged**